test/factories/license_license_permissions.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :license_license_permission do
+    license_id 0
+    license_permission_id 0
+
+    # factory :permitted_license_permission do
+    #   status LicensePermissionStatus.statuses['permitted']
+    # end
+
+    # factory :forbidden_license_permission do
+    #   status LicensePermissionStatus.statuses['forbidden']
+    # end
+
+    # factory :required_license_permission do
+    #   status LicensePermissionStatus.statuses['required']
+    # end
```

Comment:
Do we not need these commented out special factories?

Reply:
I'll remove the commented lines